lib/rubocop/cop/mixin/multiline_literal_brace_layout.rb

```diff
@@ -69,13 +69,18 @@ def check_symmetrical(node)
       end
 
       def ignored_literal?(node)
-        implicit_literal?(node) || empty_literal?(node) || node.single_line?
+        implicit_literal?(node) || empty_literal?(node) || single_line?(node)
       end
 
       def implicit_literal?(node)
         !node.loc.begin
       end
 
+      def single_line?(node)
+        return true if node.single_line?
```

Comment:
The tricky thing is that we have this module `MultilineLiteralBraceLayout` which is used in several cops (`MultilineArrayBraceLayout`, `MultilineHashBraceLayout`, `MultilineMethodCallBraceLayout`, and `MultilineMethodDefinitionsBraceLayout`), and they all check nodes of a different type in the same way. I can update it to isolate the send-specific behavior to the `MultilineMethodCallBraceLayout` cop... wondering if you had something different in mind